mailpoet/assets/js/src/automation/integrations/mailpoet/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { registerStepType } from '../../editor/store';
 import { step as SendEmailStep } from './steps/send_email';
-import { step as SomeoneSubscribedTrigger } from './steps/someone-subscribed';
+import { step as SomeoneSubscribedTrigger } from './steps/someone-subscribes';
```

Comment:
We should probably also rename `SomeoneSubscribedTrigger` to `SomeoneSubscribesTrigger`.

Reply:
Addressed in e8a1f5dd7